packages/react-components/react-virtualizer/src/components/Virtualizer/useVirtualizer.ts

```diff
@@ -101,9 +101,11 @@ export function useVirtualizer_unstable(props: VirtualizerProps): VirtualizerSta
       setIsScrolling(true);
     }
     if (scrollTimer.current) {
+      // eslint-disable-next-line no-restricted-globals
       clearTimeout(scrollTimer.current);
     }
-    scrollTimer.current = setTimeout(() => {
+    // eslint-disable-next-line no-restricted-globals
+    scrollTimer.current = window.setTimeout(() => {
```

Comment:
Using `window.setTimeout()` allows `tsc` to correct infer the return type of `number`. [The real fix is here](https://github.com/microsoft/fluentui/issues/31095).